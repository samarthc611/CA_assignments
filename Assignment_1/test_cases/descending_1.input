	.data
a:
	5
	78
	5
	2
	2
	98
	45
	36
n:
	8