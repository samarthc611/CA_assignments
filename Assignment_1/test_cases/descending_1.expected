Main Memory Contents:

0		: 98
1		: 78
2		: 45
3		: 36
4		: 5
5		: 5
6		: 2
7		: 2
